include/inria_wbc/behaviors/franka_circular_cartesian.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef IWBC_SWITCH_FOOT
+#define IWBC_SWITCH_FOOT
+#include <chrono>
+#include <iostream>
+#include <signal.h>
+
+#include <inria_wbc/behaviors/behavior.hpp>
+#include <inria_wbc/controllers/pos_tracker.hpp>
+#include <inria_wbc/estimators/cop.hpp>
+#include <inria_wbc/utils/trajectory_handler.hpp>
+
+namespace inria_wbc {
+    namespace behaviors {
+
+        class CircCartTraj : public Behavior {
```

Comment:
Does this work on the Talos too? if we put the name of the body to track in the yaml, we could apply it to the arm of the Talos too. The general idea is that ideally behaviors should not depend on the controller (as much as possible: we cannot do a squat with a Franka... but we could on a Tiago!).

Reply:
In princilpe, yes: it can be applied to any arm. 
For now, in the update section, there is specified the name of the task specific of the franka "ee" (end-effector).
But, after changing that to the desired task, then the rest is general enough to be re-used on any SE(3) task.

Reply:
can we put the name of the task in the yaml of the behavior?

Reply:
Done, I choose the name  "target_task_name"   for it